### ISSUE TITLE
CA1062 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [dotnet/roslyn-analyzers#2632]
```cs
    public partial class App
    {
        protected override void OnStartup(StartupEventArgs e)
        {
            if (e?.Args.Length == 1)
            {
                var window = e.Args[0];
                this.StartupUri = new Uri($"Windows/{window}.xaml", UriKind.Relative);
            }

            base.OnStartup(e);
        }
    }
```

**Comment by mavasani** [comment]
Verified this repros in 2.9.3, but not on https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers/2.9.4-beta1.final

**Comment by mavasani** [comment]
Duplicate of #2526
